scalyr_agent/util.py

```diff
@@ -337,18 +345,20 @@ def create_unique_id():
         is also encoded so that is safe to be used in a web URL.
     @rtype: str
     """
-    return base64.urlsafe_b64encode(sha1(uuid.uuid1().bytes).digest())
+    # 2->TODO this function should return unicode.
+    base64_id = base64.urlsafe_b64encode(sha1(uuid.uuid1().bytes).digest())
+    return base64_id.decode("utf-8")
```

Comment:
Should the documented return type be byates?

Reply:
No... 
Also, I have made changes to other docstrings.